lib/omnibus/software.rb

```diff
@@ -585,11 +585,15 @@ def with_standard_compiler_flags(env = {}, opts = {})
     # for the platform is used to join the paths.
     #
     # @param [Hash] env
+    # @param [Hash] opts
+    #   :msys => true  add the embedded msys path if building on windows.
     #
     # @return [Hash]
     #
-    def with_embedded_path(env = {})
-      path_value = prepend_path("#{install_dir}/bin", "#{install_dir}/embedded/bin")
+    def with_embedded_path(env = {}, opts = {})
```

Comment:
Needed a way to add msys paths after embedded/bin but before other stuff like git (which also has bash/less/etc.).